xonsh/completer.py

```diff
@@ -133,7 +131,7 @@ def bash_complete(self, prefix, line, begidx, endidx):
         fnme = self.bash_complete_files.get(cmd, None)
         if func is None or fnme is None:
             return set()
-        idx = 0
+        idx = n = 0
```

Comment:
What is the rationale behind this change?  The loop that follows will assign `n` anyway, no?